common/text/tree_utils_test.cc

```diff
@@ -1436,7 +1436,7 @@ TEST(GetSubtreeAsNodeTest, ValidatedFoundNodeEnumChildMismatches) {
 TEST(GetSubtreeAsLeafTest, ValidatedFoundLeaf) {
   auto root = TNode(FakeEnum::kZero, Leaf(7, "foo"));
   const auto& leaf = GetSubtreeAsLeaf(*root, FakeEnum::kZero, 0);
```

Comment:
s/&/*/

Reply:
Done (and use fully qualified type instead of auto)